gems/bess/lib/commands/houdini/nonprofit/create/create_command.rb

```diff
@@ -1,38 +1,62 @@
 module Houdini
 	module Nonprofit
 		class CreateCommand < Rails::Command::Base
-		
 			desc "Create a new nonprofit on your Houdini instance"
-			option :super_user, aliases: '-su', desc: "Make the nonprofit admin a super user (they can access any nonprofit's dashboards)"
-			option :dont_confirm_admin, default: false, type: :boolean, desc: "Require the nonprofit admin to be confirmed via email"
+			option :super_admin, aliases: '-su', default: false, type: :boolean, desc: "Make the nonprofit admin a super user (they can access any nonprofit's dashboards)"
+			option :confirm_admin, default: true, type: :boolean, desc: "Require the nonprofit admin to be confirmed via email"
+
+			option :nonprofit_name, default: nil, desc: "Provide the nonprofit's name"
+			option :state_code, default: nil, desc: "Provide the nonprofit' state code"
+			option :city, default: nil, desc: "Provide the nonprofit's city"
+			option :nonprofit_website, default: nil, desc: "[OPTIONAL] Provide the nonprofit public website"
+			option :nonprofit_email, default: nil, desc: "[OPTIONAL] Provide the nonprofit public email"
+			option :nonprofit_phone, default: nil, desc: "[OPTIONAL] Provide the nonprofit's admin's phone"
```

Comment:
Make sure to change the description since this is the nonprofit's phone, not the admin's phone.